src/app/App.tsx

```diff
@@ -52,15 +52,35 @@ const Main = ({ syncInterval }: Props) => {
 			setShowSplash(false);
 		};
 
+		const syncWallets = async () => {
+			console.log("Running wallet data sync...");
+			const profiles = env.profiles().values();
+
+			for (const profile of profiles) {
+				const [wallet] = profile.wallets().values();
```

Comment:
What's the intention of only syncing the first wallet of each profile?

Reply:
Just debug since I was having g issues to successfully sync the wallet votes, I'm gonna replace this to an iteration over profile wallets.